pkg/cmd/record/create.go

```diff
@@ -107,6 +107,7 @@ func NewCreateCommand(cfgPath *string) *cobra.Command {
 	cmd.Flags().IntVarP(&multiOpts.Threads, "parallel", "P", 4, "number of uploads (could be part) in parallel")
 	cmd.Flags().StringVarP(&multiOpts.PartSize, "part-size", "s", "128Mib", "each part size")
 	cmd.Flags().DurationVar(&timeout, "response-timeout", 5*time.Minute, "server response time out")
+	cmd.Flags().BoolVar(&multiOpts.NoTTY, "no-tty", false, "disable interactive mode for headless environments")
```

Comment:
Proposal: auto–enable `--no-tty` when the CLI is running
non-interactively.

• If the environment contains either `CI=true` or `TERM=dumb`,
  we assume no real TTY is available and flip `--no-tty` on
  automatically.

• An explicit `--tty/--no-tty` flag always overrides the
  auto-detection logic.